### DISCUSSION TITLE
leafmap is not define in jupyter lab

### DISCUSSION THREAD

**Discussion by SHsabbir25** [giswqs/geog-312#196] [Q&A] [answered]
![Screenshot 2024-12-20 150855](https://github.com/user-attachments/assets/b7d5db95-f0c6-4f63-b713-dc133a0d3853)
![Screenshot 2024-12-20 151623](https://github.com/user-attachments/assets/7e860128-4b2e-40d0-9570-f2d758306ff0)

**Comment by giswqs** [comment, marked as answer]
Uncomment the first code block and run it to install packages
